src/main/java/com/cnu2016/assignment02/SmartHome.java

```diff
@@ -0,0 +1,116 @@
+package com.cnu2016.assignment02;
+
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class SmartHome {
+    public static void main(String... args) {
+        AirConditioner airConditioner = new AirConditioner();
+        ArrayList<Appliance> list = new ArrayList<Appliance>();
```

Comment:
Why list coming in between, AirConditioner and WaterHeater ..there should be some logical flow.